packages/util-endpoints/src/utils/evaluateTemplate.ts

```diff
@@ -0,0 +1,38 @@
+// Do not remove getAttr import. It's called during template evaluation.
+import { getAttr } from "../lib";
```

Comment:
I think we could add a line like `assert getAttr` to prevent automated removal of this by bundlers or otherwise

Reply:
The comment is no longer needed, as `getAttr` is explicitly called while populating `attrShortHandMap`.

The prior version of the file attempted to use `getAttr` without actual import, and I noticed it didn't work while adding unit tests. The code was updated in commit [`45c866e` (#3884)](https://github.com/aws/aws-sdk-js-v3/pull/3884/commits/45c866ef3b3d50a3481f17b7b729c0f417a70d70)

Removed it in follow-up PR in [`af1a31b` (#3908)](https://github.com/aws/aws-sdk-js-v3/pull/3908/commits/af1a31b2a4d6daa6f69daf026828ceb913c5fdf2)